providers/darwin/host_darwin.go

```diff
@@ -20,11 +20,13 @@
 
 package darwin
 
+import "C"
```

Comment:
I realize this is a draft, but wanted to mention that the providers need to continue to function without CGO even if it is in a degraded manner (like not reporting the FQDN). Checkout https://github.com/elastic/go-sysinfo#goos--goarch-pairs.